client/js/components/GameView/Card.vue

```diff
@@ -194,4 +194,15 @@ export default {
     opacity: 0;
   }
 }
+
+@media (max-width: 600px) {
```

Comment:
I would love to see us standardize these widths for our breakpoints and move to a mixin style approach with name based widths (small, medium, large) or similar.

Reply:
Seconded. I think that can reasonably be a separate issue. Thoughts @seriouslysean?

Reply:
No need in this PR. Out of scope :)

Reply:
Also agreed, I can write an issue for that this week to take care of it